stacks_queues/queue.py

```diff
@@ -23,40 +23,78 @@ def enqueue(self, element):
             In the store are occupied
             returns None
         """
-        pass
+        # if self.buffer_size > 20:
+        #     raise QueueFullException()
+        is_added = False
+        for i in range(len(self.store)):
+            if self.store[i] is None:
+                self.store[i] = element
+                is_added = True
+                break
+        if not is_added:
+            raise QueueFullException
+        
 
     def dequeue(self):
         """ Removes an element from the Queue
             Raises a QueueEmptyException if 
             The Queue is empty.
             returns None
         """
-        pass
+        if self.empty():
+            raise QueueEmptyException
+        val = self.store[0]
+        self.store.pop(0)
+        self.store.append(None)
+        return val
+        
+        
+
 
     def front(self):
         """ Returns an element from the front
             of the Queue and None if the Queue
             is empty.  Does not remove anything.
         """
-        pass
+        return self.store[0]
         
 
     def size(self):
```

Comment:
👍 You could also maintain an attribute `size` and just increase and decrease it when new elements are added.